src/Nethermind/Nethermind.Core/Eip4844Constants.cs

```diff
@@ -5,15 +5,53 @@
 
 namespace Nethermind.Core;
 
+/// <summary>
+/// Represents the <see href="https://eips.ethereum.org/EIPS/eip-4844#parameters">EIP-4844</see> parameters.
+/// </summary>
 public class Eip4844Constants
 {
+    /// <summary>
+    /// Gets the <c>GAS_PER_BLOB</c> parameter.
+    /// </summary>
+    /// <remarks>Defaults to 2e17.</remarks>
+    public const ulong GasPerBlob = 1 << 17;
+
     public const int MinBlobsPerTransaction = 1;
 
-    public const ulong BlobGasPerBlob = 1 << 17;
-    public const ulong TargetBlobGasPerBlock = BlobGasPerBlob * 3;
-    public const ulong MaxBlobGasPerBlock = BlobGasPerBlob * 6;
-    public const ulong MaxBlobGasPerTransaction = MaxBlobGasPerBlock;
+    /// <summary>
+    /// Gets the <c>BLOB_GASPRICE_UPDATE_FRACTION</c> parameter.
+    /// </summary>
+    /// <remarks>Defaults to 3338477.</remarks>
+    public static UInt256 BlobGasPriceUpdateFraction { get; private set; } = 3338477;
+
+    /// <summary>
+    /// Gets the <c>MAX_BLOB_GAS_PER_BLOCK</c> parameter.
+    /// </summary>
+    /// <remarks>Defaults to 786432.</remarks>
+    public static ulong MaxBlobGasPerBlock { get; private set; } = GasPerBlob * 6;
+
+    public static ulong MaxBlobGasPerTransaction => MaxBlobGasPerBlock;
+
+    /// <summary>
+    /// Gets the <c>MIN_BLOB_GASPRICE</c> parameter, in wei.
+    /// </summary>
+    /// <remarks>Defaults to 1.</remarks>
+    public static UInt256 MinBlobGasPrice { get; private set; } = 1;
+
+    /// <summary>
+    /// Gets the <c>TARGET_BLOB_GAS_PER_BLOCK</c> parameter.
+    /// </summary>
+    /// <remarks>Defaults to 393216.</remarks>
+    public static ulong TargetBlobGasPerBlock { get; private set; } = GasPerBlob * 3;
+
+    // The parameter mutators are kept separate deliberately to ensure no accidental value changes.
+    #region Mutators
+    public static void OverrideBlobGasPriceUpdateFraction(UInt256 value) => BlobGasPriceUpdateFraction = value;
+
+    public static void OverrideMaxBlobGasPerBlock(ulong value) => MaxBlobGasPerBlock = value;
+
+    public static void OverrideMinBlobGasPrice(UInt256 value) => MinBlobGasPrice = value;
 
-    public static readonly UInt256 BlobGasUpdateFraction = 3338477;
-    public static readonly UInt256 MinBlobGasPrice = 1;
+    public static void OverrideTargetBlobGasPerBlock(ulong value) => TargetBlobGasPerBlock = value;
```

Comment:
Does it make sense for the mutators to be granular or maybe we should have just one taking all parameters?

Reply:
I also was thinking about this. If we move all mutators to a single method, we also move all `null` checks to that method and always call, say, `Eip4844Constants.OverrideValues(...)` in the chain spec loader. For a developer, it looks like we always override those values (until they dig into the method to see those checks), while I find it more explanatory when `null` checks are visible in the chain spec loader. So it's immediately obvious we override the values only if they are defined in the chain spec.